src/authentication.ts

```diff
@@ -12,9 +12,10 @@ const codeVerifierStorageKey = 'PKCE_code_verifier'
 const stateStorageKey = 'ROCP_auth_state'
 
 export async function redirectToLogin(config: TInternalConfig, customState?: string): Promise<void> {
-  // Create and store a random string in sessionStorage, used as the 'code_verifier'
+  // Create and store a random string in sessionStorage or localStorage, used as the 'code_verifier'
+  const storage = config.storage === 'session' ? sessionStorage : localStorage
```

Comment:
I recall me and @soofstad having a valid discussion on which properties should be in sessionStorage and which should be in localStorage, and I believe this one should default to sessionStorage (as it did.)

A fix here would be to turn the ternary around:

```suggestion
  const storage = config.storage === 'local' ? localStorage : sessionStorage
```